### DISCUSSION TITLE
Invalid model directory when running privateGPT.py

### DISCUSSION THREAD

**Discussion by joseberlines** [imartinez/privateGPT#712] [Q&A]
The follwoing is a situation that I get after trying to run the privateGPT.py file:

![image](https://github.com/imartinez/privateGPT/assets/33203439/48cac58c-86b5-42ef-b511-4aa6ca5baa88)

basically the error is:
File "/home/jxxxxxxxxx/shared/users/j/jxxxxxxxxxx/2023_06_12_G4A/privateGPT/privateGPT.py", line 37, in main
    llm = GPT4All(model=model_path, n_ctx=model_n_ctx, backend='gptj', callbacks=callbacks, verbose=True)
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for GPT4All
__root__
  Invalid model directory (type=value_error)

the models file looks like this:
![image](https://github.com/imartinez/privateGPT/assets/33203439/3fd59134-a1fa-4925-8d97-8a0005d1af97)

Does someone knows what is going on here?

I have followed for two times all the instructions of this video:
https://www.youtube.com/watch?v=jxSPx1bfl2M

thanks a lot

**Comment by sime2408** [comment]
You're running a conda environment so I suspect that you have to add a full absolute path to your model as an env variable not relative. Can you try that?
